examples/jf-control-app/main.cpp

```diff
@@ -46,6 +47,17 @@ CHIP_ERROR RpcConnect(void)
     return chip::rpc::client::StartPacketProcessing();
 }
 
+void registerJCMCommands(Commands & commands, CredentialIssuerCommands * credsIssuerConfig)
```

Comment:
I don't see this function being called. I guess it should be called at the end of this file, right before
`return commands.Run(static_cast<int>(c_args.size()), c_args.data())`